pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/medical/records/MedicalRecordDao.java

```diff
@@ -99,11 +107,29 @@ private void setFieldsToStatement(MedicalRecord medicalRecord, PreparedStatement
 
     @Override
     public int delete(int id) {
-        return 0;
+        SimpleResult simpleResult = DaoUtil.deleteById(id, SQL_DELETE_BY_ID);
```

Comment:
Similar thought from ddp_institution.  ddp_medical_record is used in many join statements outside of this code. Any impact to those if one of these is deleted after it's been used by one of those "joining" queries?